clients/admin-ui/src/features/common/custom-fields/form/CustomInput.tsx

```diff
@@ -17,6 +18,21 @@ import { FieldHookConfig, useField } from "formik";
 
 import { InputType } from "./types";
 
+/**
+ * This isn't the cleanest way to style these forms, but it's a quick hack. Ideally our form inputs
+ * (like CustomSelect) would all use the same styles, or at least have a limited set of variants:
+ * https://chakra-ui.com/docs/styled-system/component-style#styling-multipart-components
+ *
+ * Unfortunately the designs for these forms call for styles that are inconsistent with the other
```

Comment:
Has this concern been brought to Design?

Reply:
Here's some screenshots of the difference.

How most of our labels look:
<img width="472" alt="Screen Shot 2023-02-10 at 10 36 16" src="https://user-images.githubusercontent.com/2236777/218171250-05fdc333-303c-4402-914d-057ab4076879.png">

How the labels in the modal look (smaller, bolder, grayer):
<img width="388" alt="Screen Shot 2023-02-10 at 10 36 24" src="https://user-images.githubusercontent.com/2236777/218171244-5bf0e633-9d72-42da-a985-6db228111231.png">